code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -96,7 +96,7 @@
 	icon_living = icon_state
 	icon_dead = "[visualAppearence]_dead"
 
-/obj/effect/mob_spawn/drone/dusty
+/obj/effect/mob_spawn/drone/derelict
 	name = "derelict drone shell"
 	desc = "A long-forgotten drone shell. It seems kind of... Space Russian."
```

Comment:
is it long forgotten if its straight out of the drone lathe

Reply:
I dont care. Deal with it.